create_rental.rb

```diff
@@ -0,0 +1,31 @@
+require_relative 'list_books'
+require_relative 'list_people'
+
+class CreateRental
+  def initialize(state)
+    @book_list = state[:book_list]
+    @people_list = state[:people_list]
+    @rental_list = state[:rental_list]
+    @list_books = ListBooks.new(state)
+    @list_people = ListPeople.new(state)
+  end
+
+  def ask_user_input
+    p 'Select a book from the following list by number'
+    @list_books.list_books_with_index
+    book = @book_list[gets.chomp.to_i]
+    p 'Select a person from the following list by number (not id)'
+    @list_people.list_people_with_index
+    person = @people_list[gets.chomp.to_i]
+    print 'Date: '
+    date = gets.chomp
+    [book, person, date]
+  end
```

Comment:
- [ ] When I try to create a rental, I get this error in the terminal:

![image](https://user-images.githubusercontent.com/77106962/180850511-aaf376d0-15ab-48f6-8a67-8e5bad305475.png)
 
#### There are no instance methods or instance variables called `list_books_with_index` and `list_people_with_index` in the ListBooks and ListPeople classes. Kindly recheck. 👍